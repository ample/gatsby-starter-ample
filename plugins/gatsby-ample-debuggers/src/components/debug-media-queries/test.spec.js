import React from "react"
import renderer from "react-test-renderer"

import DebugMediaQueries from "./"

describe("DebugMediaQueries", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<DebugMediaQueries isShowing={"true"} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
