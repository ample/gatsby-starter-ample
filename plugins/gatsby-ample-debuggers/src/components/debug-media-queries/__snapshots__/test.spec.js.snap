// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DebugMediaQueries renders correctly 1`] = `
<div
  className="debug"
  onClick={[Function]}
>
  <div
    className="debug_media_queries"
  />
</div>
`;
